src/bio/bio_xstream.c

```diff
@@ -31,7 +32,7 @@
 /* SPDK blob parameters */
 #define DAOS_BS_CLUSTER_SZ	(1ULL << 25)	/* 32MB */
 /* DMA buffer parameters */
-#define DAOS_DMA_CHUNK_INIT_PCT	60	/* Default pre-xstream init chunks, in percentage */
+#define DAOS_DMA_CHUNK_INIT_PCT 50      /* Default pre-xstream init chunks, in percentage */
```

Comment:
Should this read "per-xstream"?
If we are seeing out of memory when trying to grow the buffer, why does it help only allocating 50% initially, won't we end up seeing the same out of memory issues after multiple buffer growths?

Reply:
yes, sorry for the typo.

I was thinking that DAOS engine usually start once server node started, and we'd try to hold hugepages as much as possible when the memory isn't fragmented and no other potential hugepage consumers.